docs/hub/webhooks.md

```diff
@@ -0,0 +1,193 @@
+
+For example, if the `"repo.config.dois"` scope is added in the future, any event with that scope can be considered by your webhook handler as an `"update"` action on the `"repo.config"` scope.
+
+### Repo
+
+In the current version of webhooks, the top level property `repo` is always specified, as events can always be associated to a repo. For example, consider the following value:
+
+```json
+"repo": {
+	"type": "model",
+	"name": "some-user/some-repo",
+	"id": "6366c000a2abcdf2fd69a080",
+	"private": false,
+	"url": {
+		"web": "http://huggingface.co/some-user/some-repo",
+		"api": "http://huggingface.co/api/models/some-user/some-repo"
+	},
+	"headSha": "c379e821c9c95d613899e8c4343e4bfee2b0c600",
+	"tags": [
+		"license:other",
+		"has_space"
+	],
+	"owner": {
+		"id": "61d2000c3c2083e1c08af22d"
+	}
+}
+```
+
+`repo.headSha` is the sha of the latest commit on the repo's `main` branch. It is only sent when `event.scope` starts with `"repo"`, not on community events like discussions and comments.
+
+### Discussion
+
+The top level property `discussion` is specified on community events. The `discussion.isPullRequest` property is a boolean indicating if the discussion is also a pull request (on HF, a PR is a special case of a Discussion). Here is an example value:
+
+```json
+"discussion": {
+	"id": "639885d811ae2bad2b7ba461",
+	"title": "Hello!",
+	"url": {
+		"web": "https://huggingface.co/some-user/some-repo/discussions/3",
+		"api": "https://huggingface.co/api/models/some-user/some-repo/discussions/3"
+	},
+	"status": "open",
+	"author": {
+		"id": "61d2000c3c2083e1c08af22d"
+	},
+	"isPullRequest": true,
+	"changes": {
+		"base": "refs/heads/main"
+	}
+	"num": 3
+}
+```
+
+### Comment
+
+The top level property `comment` is specified when a comment is created (including on discussion creation) or updated. Here is an example value:
+
+```json
+"comment": {
+	"id": "6398872887bfcfb93a306f18",
+	"author": {
+		"id": "61d2000c3c2083e1c08af22d"
+	},
+	"content": "This adds an env key",
+	"hidden": false,
+	"url": {
+		"web": "http://huggingface.co/some-user/some-repo/discussions/4#6398872887bfcfb93a306f18"
+	}
+}
+```
+
+## Webhook secret
+
+Setting a webhook secret is useful to make sure payloads sent to your webhook handler come from Hugging Face.
+
+If you set a secret for your webhook, it will be sent along as an `X-Webhook-Secret` HTTP header on every request. Only ASCII characters are supported.
+
+<Tip>
+You can also change the URL of the webhook to add a secret to the URL. For example, setting it to `https://example.com/webhook?secret=XXX`.
+
+This can be helpful if accessing the HTTP headers of the request is complicated for your webhook handler.
+</Tip>
+
+## Rate limiting
+
+Each webhook is limited to 1,000 triggers per 24 hours. You can check the  daily triggers for your webhook in your webhook settings, in the "Activity" tab.
```

Comment:
Maybe add a link here for the activity tab

Reply:
not possible - the activity tab is hf.co/settings/webhooks/xxxx/activity , it depends on the individual webhook's id